testing/ppsspp/APKBUILD

```diff
@@ -0,0 +1,82 @@
+# Contributor: Díaz Urbaneja Diego <sodomon2@gmail.com>
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=ppsspp
+pkgver=1.8.0
+pkgrel=0
+pkgdesc="PPSSPP - a fast and portable PSP emulator"
+url="https://www.ppsspp.org/"
+arch="x86_64"
+license="GPL-2.0-only"
+depends="cmake make"
+makedepends="glew-dev libzip-dev snappy-dev mesa-dev qt5-qtbase-dev zlib-dev sdl2-dev"
+options="!check"
+source="$pkgname-$pkgver.tar.gz::https://github.com/hrydgard/$pkgname/archive/v$pkgver.tar.gz
+  $pkgname-$pkgver-armips.zip::https://github.com/Kingcom/armips/archive/7885552b208493a6a0f21663770c446c3ba65576.zip
+  $pkgname-$pkgver-discord-rpc.zip::https://github.com/discordapp/discord-rpc/archive/3d3ae7129d17643bc706da0a2eea85aafd10ab3a.zip
+  $pkgname-$pkgver-glslang.zip::https://github.com/hrydgard/glslang/archive/f9d08a25fbe17e0677a89d398f4d7f232339c3f9.zip
+  $pkgname-$pkgver-SPIRV-Cross.zip::https://github.com/KhronosGroup/SPIRV-Cross/archive/be7425ef70231ab82930331959ab487d605d0482.zip
+  $pkgname-$pkgver-rapidjson.zip::https://github.com/Tencent/rapidjson/archive/73063f5002612c6bf64fe24f851cd5cc0d83eef9.zip
+  $pkgname-$pkgver-lang.zip::https://github.com/hrydgard/$pkgname-lang/archive/03707d64bffc594d5707780a34ada77e8f43cdc0.zip
+  $pkgname-$pkgver-ffmpeg.zip::https://github.com/hrydgard/$pkgname-ffmpeg/archive/7472c903ec771c00af9925097e8d37075e9a379f.zip
```

Comment:
We can't use our system libs here? :/

Reply:
we can't because they use modified libraries

Reply:
@Cogitri  that's true.. upstream has modified libs due the emulation requirements.. in fact it is quite an achievement to have made walk this emulator in the already very different alpine alpine.